### DISCUSSION TITLE
[General]: Browser Support

### DISCUSSION THREAD

**Discussion by Gh-Moradi** [davidjerleke/embla-carousel#1026] [General]
### Summary

Hello, Does Embla-Carousel support all browsers like Glide.js?
https://github.com/glidejs/glide?tab=readme-ov-file#browser-support

### If applicable, which variants of Embla Carousel are relevant to this conversation?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by Gh-Moradi** [comment]
Can Embla support of these browsers?

1. IE 11+
2. Edge
3. Chrome 10+
4. Firefox 10+
5. Opera 15+
6. Safari 5.1+
7. Safari iOS 9+

I searched the documents and found nothing!

**Reply by sarussss** [reply]
Hi @Gh-Moradi 
I think it is very difficult for an individual to give the most accurate answer, instead I hope everyone who is using Embla if they see this question, please take a moment to comment the browser you are using and its version. From there, there will be enough information to give the most accurate answer.
I am using:
Safari 16.1 
Safari iOS 17.5.1
Chrome 129.0.6668.90

**Reply by sarussss** [reply]
As an added bonus, Embla uses ResizeObserverEntry so we can base at least that the browser must support ResizeObserverEntry to be able to work well with Embla.
https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserverEntry#browser_compatibility

**Comment by davidjerleke** [comment]
@Gh-Moradi please [read this](https://github.com/davidjerleke/embla-carousel/issues/11#issuecomment-2312002787).

In addition to that, anyone is welcome to share if they’ve tested Embla on a specific browser and version here.
